src/formulations/inverse-dynamics-formulation-acc-force.cpp

```diff
@@ -116,7 +116,7 @@ bool InverseDynamicsFormulationAccForce::addMotionTask(TaskMotion & task,
 {
   assert(weight>=0.0);
   assert(transition_duration>=0.0);
-  
+
```

Comment:
This file should not be changed in this PR

Reply:
Done, I think (let me know if there is an issue here, as I am not 100% sure if I did that correctly).